compiler/rustc_target/src/spec/targets/arm64e_apple_ios.rs

```diff
@@ -0,0 +1,30 @@
+use crate::spec::base::apple::{ios_llvm_target, opts, Arch};
+use crate::spec::{FramePointer, SanitizerSet, Target, TargetOptions};
+
+pub fn target() -> Target {
+    let arch = Arch::Arm64e;
+    let mut base = opts("ios", arch);
+    base.supported_sanitizers = SanitizerSet::ADDRESS | SanitizerSet::THREAD;
+
+    Target {
+        llvm_target: ios_llvm_target(arch).into(),
+        pointer_width: 64,
+        data_layout: "e-m:o-i64:64-i128:128-n32:64-S128".into(),
+        arch: arch.target_arch(),
+        options: TargetOptions {
+            features: "+neon,+fp-armv8,+apple-a12,+v8.3a,+paca,+pacg".into(),
+            max_atomic_width: Some(128),
+            forces_embed_bitcode: true,
+            frame_pointer: FramePointer::NonLeaf,
+            bitcode_llvm_cmdline: "-triple\0\
```

Comment:
While its not a Git conflict, all of the legacy bitcode stuff was removed from the other targets last night in https://github.com/rust-lang/rust/pull/117364. I can open a follow-up PR to remove it here as well if you don't want to do it since the timing is kinda awkward.

Reply:
If you could open a followup PR, that would be awesome :) I'd rather not have @arttet wait on another round of review (i.e. me)

Reply:
Will do. I'll watch for this one to merge then followup.